### PR TITLE
[hi] Localize /en/docs/reference/glossary/replica-set.md

### LINKED ISSUE #37520: [hi] Localize /en/docs/reference/glossary/replica-set.md
### Prerequisites:

- If this is your first contribution, visit [Contribute to K8s docs](https://kubernetes.io/docs/contribute/) for information on how to get started.
- Review a few https://github.com/kubernetes/website/issues/35600#issuecomment-1214215859 to see which 
  content you have to change and which you leave unchanged.

### Procedure:

- Make sure your base, target branch is `main` and create a working branch called `hi-replica-set.md`.
- Create a file called content/hi/docs/reference/glossary/replica-set.md
- Copy the content of 
  https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/replica-set.md to 
  the file you created
- Localize the content.
- Push your change and create a pull request.

Completing this feature issue fixes part of https://github.com/kubernetes/website/issues/35600

> **Note**:- Don't forget to join [`slack`](https://communityinviter.com/apps/kubernetes/community) `kubernetes-docs-hi` channel if facing any problem!
 

/language hi
/triage accepted

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
### Fixes

Closes #37520

### Chnages proposed

- Localize the [K8-website/blob/main/content/en/docs/reference/glossary/replica-set.md](https://github.com/Pradumnasaraf/K8-website/blob/main/content/en/docs/reference/glossary/replica-set.md) in Hindi

If any changes are required let me know, thank you.

### CODE REVIEW

File: content/hi/docs/reference/glossary/replica-set.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- fundamental
+- core-object
+- workload
+---
+ एक रेप्लिकासेट (उद्देश्य) किसी भी समय चल रहे रेप्लिका पॉड्स के एक सेट को बनाए रखता है।
+
+<!--more-->
+
+वर्कलोड की ऑब्जेक्ट जैसे {{< glossary_tooltip term_id="deployment" >}} रेप्लिकासेट्स का उपयोग करें
+यह सुनिश्चित करने के लिए कि कॉन्फ़िगर की गई संख्या {{< glossary_tooltip term_id="pod" text="पॉड्स" >}}
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/replica-set.md:20]:
```suggestion
यह सुनिश्चित करने के लिए करते हैं कि आपके क्लस्टर में {{< glossary_tooltip term_id="pod" text="पॉड्स" >}} की कॉन्फ़िगर की गई संख्या चल रही है, उस रेप्लिकासेट के विनिर्देश के आधार पर।
```

File: content/hi/docs/reference/glossary/replica-set.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- core-object
+- workload
+---
+ एक रेप्लिकासेट (उद्देश्य) किसी भी समय चल रहे रेप्लिका पॉड्स के एक सेट को बनाए रखता है।
+
+<!--more-->
+
+वर्कलोड की ऑब्जेक्ट जैसे {{< glossary_tooltip term_id="deployment" >}} रेप्लिकासेट्स का उपयोग करें
+यह सुनिश्चित करने के लिए कि कॉन्फ़िगर की गई संख्या {{< glossary_tooltip term_id="pod" text="पॉड्स" >}} 
+आपके क्लस्टर में चल रहे हैं, उस रेप्लिकासेट के विनिर्देश के आधार पर।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/replica-set.md:21]:
```suggestion
```

File: content/hi/docs/reference/glossary/replica-set.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: रेप्लिकासेट (ReplicaSet)
+id: replica-set
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/replicaset/
+short_description: >
+  रेप्लिकासेट सुनिश्चित करता है कि एक समय में एक निर्दिष्ट संख्या में पॉड की प्रतिकृतियां चल रही हैं।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/replica-set.md:7]: I think that this way the sentence is little easier to understand. Thoughts, WDYT @Pradumnasaraf ? (I do want other contributors/reviewers to have a look at this suggestion too)
```suggestion
  रेप्लिकासेट सुनिश्चित करता है कि पॉड प्रतिकृतियों की एक निर्दिष्ट संख्या एक समय में चल रही हैं।
```

**Review by Babapool — COMMENTED** [review]
Hope you find these suggestions useful. Thanks:

**Review by Pradumnasaraf — COMMENTED** [review]
Hey, @Babapool I have done the changes recommend by you. Can you have a look?
Thank you.

**Review by Babapool — APPROVED** [review]
These changes LGTM now. (I am not an official `hi` reviewer/approver). Thanks

**Comment by Babapool** [comment]
Hey @Pradumnasaraf , can you push an empty commit so that it can trigger the rebuild checks. Thanks

**Comment by Pradumnasaraf** [comment]
Ready for review, thanks!

**Review by verma-kunal — APPROVED** [review]
lgtm 👍
Thanks for reviewing @Babapool

**Review by bishal7679 — APPROVED** [review]
The changes LGTM!

File: content/hi/docs/reference/glossary/replica-set.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+- fundamental
+- core-object
+- workload
+---
+ एक रेप्लिकासेट (का उद्देश्य) किसी भी समय चल रहे रेप्लिका पॉड्स का एक समूह बनाए रखता है।
+
+<!--more-->
+
+वर्कलोड ऑब्जेक्ट्स जैसे {{< glossary_tooltip term_id="deployment" >}} रेप्लिकासेट्स का उपयोग
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/replica-set.md:19]: You need to insert deployment written in the Devnagari script here. It is missing in the Glossary tooltip text section.
```suggestion
वर्कलोड ऑब्जेक्ट्स जैसे {{< glossary_tooltip text="" term_id="deployment" >}} रेप्लिकासेट्स का उपयोग
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

**Comment by Pradumnasaraf** [comment]
Thank you for reviewing, @divya-mohan0209. I have done the changes suggested by you and also squashed the commits.

File: content/hi/docs/reference/glossary/replica-set.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+- fundamental
+- core-object
+- workload
+---
+ एक रेप्लिकासेट (का उद्देश्य) किसी भी समय चल रहे रेप्लिका पॉड्स का एक समूह बनाए रखता है।
+
+<!--more-->
+
+वर्कलोड ऑब्जेक्ट्स जैसे {{< glossary_tooltip text="" term_id="deployment" >}} रेप्लिकासेट्स का उपयोग
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/replica-set.md:19]: @Pradumnasaraf Please make these changes. I do not see them reflected.
```suggestion
वर्कलोड ऑब्जेक्ट्स जैसे {{< glossary_tooltip text="डिप्लॉइमन्ट" term_id="deployment" >}} रेप्लिकासेट्स का उपयोग
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

**Comment by Pradumnasaraf** [comment]
So sorry for that, there was some error while force pushing. Fixed that. 
Thank you, @divya-mohan0209

**Comment by Pradumnasaraf** [comment]
Hey, @divya-mohan0209 can you please have a look and let me know if any changes are required? Thank you

**Comment by Pradumnasaraf** [comment]
Hey, @divya-mohan0209 can you please have a look and let me know if any changes are required? Thank you

File: content/hi/docs/reference/glossary/replica-set.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+---
+title: रेप्लिकासेट (ReplicaSet)
+id: replica-set
+date: 2018-04-12
+full_link: /docs/concepts/workloads/controllers/replicaset/
+short_description: >
+  रेप्लिकासेट सुनिश्चित करता है कि पॉड प्रतिकृतियों की एक निर्दिष्ट संख्या एक समय में चल रही हैं।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/replica-set.md:7]:
```suggestion
  रेप्लिकासेट यह  सुनिश्चित करता है कि किसी एक अवसर पर निर्दिष्ट संख्या में पॉड प्रतिकृतियां चल रही हैं।
```

File: content/hi/docs/reference/glossary/replica-set.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+- fundamental
+- core-object
+- workload
+---
+ एक रेप्लिकासेट (का उद्देश्य) किसी भी समय चल रहे रेप्लिका पॉड्स का एक समूह बनाए रखता है।
+
+<!--more-->
+
+वर्कलोड ऑब्जेक्ट्स जैसे {{< glossary_tooltip text="डिप्लॉइमन्ट" term_id="deployment" >}} रेप्लिकासेट्स का उपयोग
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/replica-set.md:19]:
```suggestion
वर्कलोड ऑब्जेक्ट्स, जैसे {{< glossary_tooltip text="डिप्लॉयमेंट" term_id="deployment" >}}, रेप्लिकासेट्स
```

File: content/hi/docs/reference/glossary/replica-set.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+यह सुनिश्चित करने के लिए करते हैं कि आपके क्लस्टर में {{< glossary_tooltip term_id="pod" text="पॉड्स" >}} की कॉन्फ़िगर की गई संख्या चल रही है, उस रेप्लिकासेट के विनिर्देश के आधार पर।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/replica-set.md:20]:
```suggestion
के विनिर्देश के आधार पर आपके क्लस्टर पर कॉन्फ़िगर की गई संख्या के {{< glossary_tooltip term_id="pod" text="पॉड्स" >}} चल रहे है ये सुनिश्चित करते हैं।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Slightly changed the grammar & simplified things. WDYT?

File: content/hi/docs/reference/glossary/replica-set.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+short_description: >
+  रेप्लिकासेट सुनिश्चित करता है कि पॉड प्रतिकृतियों की एक निर्दिष्ट संख्या एक समय में चल रही हैं।
+
+aka: 
+tags:
+- fundamental
+- core-object
+- workload
+---
+ एक रेप्लिकासेट (का उद्देश्य) किसी भी समय चल रहे रेप्लिका पॉड्स का एक समूह बनाए रखता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/replica-set.md:15]:
```suggestion
 एक रेप्लिकासेट (का उद्देश्य) किसी भी समय चल रहे रेप्लिका पॉड्स का एक समूह बनाए रखना है।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Some more nits

File: content/hi/docs/reference/glossary/replica-set.md (hunk 6 above)
Comment (Pradumnasaraf) [reply content/hi/docs/reference/glossary/replica-set.md:7]: Agree, now sentence sounds more natural and simplified, thank you

**Comment by Pradumnasaraf** [comment]
Thank you, @divya-mohan0209 for the review. I have committed to all the suggested changes by you. 
Also, thank you for making my Hindi better :)

File: content/hi/docs/reference/glossary/replica-set.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+- fundamental
+- core-object
+- workload
+---
+ एक रेप्लिकासेट (का उद्देश्य) किसी भी समय चल रहे रेप्लिका पॉड्स का एक समूह बनाए रखना है।
+
+<!--more-->
+
+वर्कलोड ऑब्जेक्ट्स, जैसे {{< glossary_tooltip text="डिप्लॉयमेंट" term_id="deployment" >}}, रेप्लिकासेट्स
+के विनिर्देश के आधार पर आपके क्लस्टर पर कॉन्फ़िगर की गई संख्या के {{< glossary_tooltip term_id="pod" text="पॉड्स" >}} चल रहे है ये सुनिश्चित करते हैं।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/replica-set.md:20]: Small grammatical nit. Though I'd like someone with more prowess in Hindi to confirm. @anubha-v-ardhan @Babapool @verma-kunal 

```suggestion
के विनिर्देश के आधार पर आपके क्लस्टर पर कॉन्फ़िगर की गई संख्या में {{< glossary_tooltip term_id="pod" text="पॉड्स" >}} चल रहे है ये सुनिश्चित करते हैं।
```

Comment (Pradumnasaraf) [reply content/hi/docs/reference/glossary/replica-set.md:20]: Done, thank you.

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

File: content/hi/docs/reference/glossary/replica-set.md (hunk 10 above)
Comment (Babapool) [reply content/hi/docs/reference/glossary/replica-set.md]: LGTM this change

**Comment by divya-mohan0209** [comment]
@Babapool @Garima-Negi @bishal7679 @anubha-v-ardhan @verma-kunal PTAL whenever free & advise if this is good to be merged?

**Review by Babapool — COMMENTED** [review]
LGTM 👍

**Review by bishal7679 — APPROVED** [review]
Looks good!
@divya-mohan0209 we can merge this one!

**Pull request merged**